.github/workflows/env-tests.yml

```diff
@@ -40,10 +40,11 @@ jobs:
           # Use yamllint to check the actual yaml files.
           yamllint .github/workflows/*.yml
           yamllint ./envs/conda_build_config.yaml
+          export PYTHONPATH=./open-ce
```

Comment:
It would be better to pip install the package, then just call `open-ce validate env` instead of putting the full path to the open-ce tool.